code/socialDistribution/views.py

```diff
@@ -919,3 +913,20 @@ def inbox(request):
     }
 
     return render(request, 'author/inbox.html', context)
+
+def public_share(request, id):
+    """
+        Renders a single public post shared via link (including unlisted public posts)
+    """
+    post = get_object_or_404(LocalPost, pk=id) 
+    author = post.author
+   
+    if not post.is_public:
```

Comment:
Friends posts are currently being called. `post.is_public` is a method so it needs to be called.

Reply:
thanks, fixed